regress/pesterTests/Uninstall.Tests.ps1

```diff
@@ -0,0 +1,235 @@
+﻿If ($PSVersiontable.PSVersion.Major -le 2) {$PSScriptRoot = Split-Path -Parent $MyInvocation.MyCommand.Path}
+Import-Module $PSScriptRoot\CommonUtils.psm1 -Force
+$suite = "Uninstall"
+$tC = 1
+$tI = 0
+Describe "Uninstall Tests" -Tags "Uninstall" {
```

Comment:
How does this work for GitHub CI test runs? We only unregister the services as part of uninstall right?

Reply:
Please see line 25, I run "uninstall-sshd.ps1" in BeforeAll. note that Uninstall.tests.ps1 only run after all tests are done